ddtrace/bootstrap/sitecustomize.py

```diff
@@ -63,6 +58,12 @@
     "pyramid": True,
 }
 
+MODULES_THAT_TRIGGER_CLEANUP_WHEN_INSTALLED = ("gevent",)
+
+MODULES_TO_NOT_CLEANUP = set("atexit", "asyncio", "attr", "concurrent", "ddtrace", "logging", "typing")
```

Comment:
```suggestion
MODULES_TO_NOT_CLEANUP = {"atexit", "asyncio", "attr", "concurrent", "ddtrace", "logging", "typing"}
```
This is apparently faster than using `set()` and doesn't break mypy 😄

Reply:
Nice, I didn't realize that syntax worked in 2.7